src/Microsoft.Azure.Functions.Extensions.Mcp/Http/ISseRequestHandler.cs

```diff
@@ -9,5 +9,5 @@ internal interface ISseRequestHandler
 {
     Task HandleRequest(HttpContext context);
 
-    bool IsSseRequest(HttpContext context);
+    bool IsLegacySseRequest(HttpContext context);
```

Comment:
Is there a non legacy sse request? Just wondering why we need to prefix this with legacy? Isn't SSE still an  option or is that getting deprecated entirely?